tests/xtro-sharpie/SimdCheck.cs

```diff
@@ -43,6 +43,8 @@ class NativeSimdInfo
 			{ "vector_uint2", new NativeSimdInfo { Managed = "Vector2i", }},
 			{ "vector_uint3", new NativeSimdInfo { Managed = "Vector3i", }},
 			{ "vector_uint4", new NativeSimdInfo { Managed = "Vector4i", }},
+			// simd_dobleX is typedefed to vector_doubleX
```

Comment:
missing `u` (at least in English and French)

Reply:
In Spanish is ok, doble :)